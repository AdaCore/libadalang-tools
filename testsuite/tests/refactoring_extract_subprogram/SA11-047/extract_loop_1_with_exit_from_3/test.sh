extract_subprogram -P default.gpr -S my_package.adb -SL 9 -SC 9 -EL 9 -EC 10 -N Foo