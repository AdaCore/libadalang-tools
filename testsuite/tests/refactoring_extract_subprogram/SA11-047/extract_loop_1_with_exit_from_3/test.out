Extracting as a ADA_SUBP_KIND_PROCEDURE:
my_package.adb
7:1-7:1 
      procedure Foo;

      procedure Foo is
      begin
         J_Loop :
         for J in Start_Index .. End_Index loop
            K_Loop :
            for K in Start_Index .. End_Index loop
               L_Loop :
               for L in Start_Index .. End_Index loop
                  exit L_Loop;
               end loop L_Loop;
            end loop K_Loop;
         end loop J_Loop;
      end Foo;


8:7-17:23 Foo;

